tensorrt_llm/_torch/modules/fused_moe/quantization.py

```
@@ -4528,3 +4528,459 @@ def create_weights(self, module: torch.nn.Module):
+            w1, w3, w2, w1_scale, w3_scale, w2_scale = get_expert(
+                weight_dict, expert_id)
+
+            # DeepGEMM expects L1 in [gate | up] order before
+            # transform_weights_for_mega_moe interleaves gate/up rows.
+            # TRT-LLM checkpoints map gate_proj -> w1 and up_proj -> w3.
+            dst_w3_w1_weight[slot_id].copy_(
+                torch.cat([
+                    self._to_weight_device_uint8(w1, dst_w3_w1_weight),
+                    self._to_weight_device_uint8(w3, dst_w3_w1_weight),
+                ],
+                          dim=0),
```

Flake8 (7.3.0)

[error] continuation line over-indented for hanging indent

(E126)